### PR TITLE
Remove founding member copy

### PR DESCRIPTION
**Pull request by davidrapson**
Founding membership ends tomorrow (31/03/2015). This PR removes all founding member copy.

### CODE REVIEW

File: frontend/app/views/info/about.scala.html [hunk 1]
```diff
@@ -100,7 +84,7 @@
     @* ===== Become a Patron ===== *@
     @for(img <- pageImages.find(_.name.contains("patrons"))) {
         <div class="page-slice l-constrained">
-            @fragments.promos.promoPrimary(title="Become a Patron and support the Guardian's future", img=img, toneClass=Some("tone-trans-brand-dark")) {
+            @fragments.promos.promoSecondary(title=Html("Become a Patron and support the Guardian's future"), img=img, toneClass=Some("tone-trans-brand-dark")) {
```
Comment (rtyley) [inline frontend/app/views/info/about.scala.html]: Why's this `Html()` declaration needed?!

Comment (davidrapson) [inline frontend/app/views/info/about.scala.html]: It isn't, but I need to refactor the `promoSecondary` items to remove it in another PR

Comment (davidrapson) [inline frontend/app/views/info/about.scala.html]: (As in `promoSecondary` currently accepts Html)

**Comment by davidrapson** [comment]
Confirmed founding member copy removed on PROD

screen shot 2015-03-31 at 11 55 37

**Pull request merged**